django_slack/api.py

```diff
@@ -135,11 +135,9 @@ def slack_message(template, context=None, attachments=None, blocks=None, fail_si
                     "specified".format(k),
                 )
 
-        if 'attachments' in data:
-            data['attachments'] = json.dumps(data['attachments'])
-        
-        if 'blocks' in data:
-            data['blocks'] = json.dumps(data['blocks'])
+        for arg in ('attachments', 'blocks',):
```

Comment:
Can I be super pedantic and ask you to use `x` as the anonymous variable and drop the trailing `,` ? :)